### ISSUE TITLE
RA - Actor type gnrl: Bogus yaml removals: ScaredyCat

### ISSUE THREAD

**Issue by MustaphaTR** [OpenRA/OpenRA#8597]
I was tring update my map to playtest. When i select bleed to check map Resource Center gives that error. Why i'm writing it there. Because it is defined infantry.yaml not my map's yaml file. It wasn't giving any error on older versions.

Map: http://resource.openra.net/maps/3470/

`-ScaredyCat` isn't there on my fork. Not sure which PR added this.

**Comment by GraionDilach** [comment]
gnrl - while inheriting ^CivInfantry - does not have a panic sequence defined for ScaredyCat which was caught and thereby removed in #7709.

This seems to be related to/a duplicate of #8086. Not sure what to do here. Best I can do is to notify @reaperrr to reorganize gnrl's inheriting in #8515.

**Comment by reaperrr** [comment]
I checked, there should be no problems once #8515 is merged unless the map itself defines another (bogus) ```-ScaredyCat```.

**Comment by MustaphaTR** [comment]
Works now.
